README.md

```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
 [**Resources**](#resources) |
 [**Organizations Using Superset**](RESOURCES/INTHEWILD.md)
 
+## Why Superset?
+
+Superset is a modern data exploration and data visualization platform. Superset can replace or augment proprietary business intelligence tools for many teams.
+
+Superset provides:
+
+- A **no-code interface** for building charts quickly
+- A powerful, web-based **SQL Editro** for advanced querying
```

Comment:
```suggestion
- A powerful, web-based **SQL Editor** for advanced querying
```

Reply:
Actually, not sure if we should capitalize Editor here.

Reply:
I vote for capitalize! It's a feature name